src/groundStation/groundStation.py

```diff
@@ -324,6 +333,11 @@ def getOptions(self):
             type=int,
             default='15000', # 15 seconds
             help='RDP connection timeout')
+        self.parser.add_argument(
+            '--satellite',
```

Comment:
```suggestion

        self.parser.add_argument(
            '-s',
            '--satellite',
```
Just to conform with other cli arguments and it makes it quicker to specify as an arg (unless it conflicts with something I don't know of)